qtpy/tests/test_qtwebenginewidgets.py

```diff
@@ -1,10 +1,14 @@
 import pytest
+from packaging import version
 
-from qtpy import PYQT6, PYSIDE6
+from qtpy import PYQT6, PYQT_VERSION, PYSIDE6, PYSIDE_VERSION
```

Comment:
Following the other comment. now we need here `PYQT5` and `PYSIDE2` constants so this should be something like:
```suggestion
from qtpy import PYQT5, PYSIDE2, PYQT6, PYQT_VERSION, PYSIDE6, PYSIDE_VERSION
```